flasksrc/quiz.py

```diff
@@ -14,6 +15,26 @@ class ToyNetQuizItem(Schema):
 class ToyNetQuizGetResp(Schema):
     items = fields.List(fields.Nested(ToyNetQuizItem))
 
+class ToyNetQuizScorePostReq(Schema):
+    quiz_id = fields.Int(required=True)
+    user_id = fields.String(required=True)
+    count_correct = fields.Int(required=True)
+    count_wrong = fields.Int(required=True)
+
+class ToyNetQuizScorePostResp(Schema):
+    submission_id = fields.Int()
+
+class ToyNetQuizScoreItem(Schema):
```

Comment:
I think we can rename `ToyNetQuizScoreItem`  to `ToyNetQuizScore`. I like how you structured this data!